### PR TITLE
Add initial narrative docs

### LINKED ISSUE #36: Add developer documentation
- [x] Add basic development docs to the README (Philip)
- [ ] Add development docs to the documentation website (Pavithra)

### PR DESCRIPTION
**Pull request by pavithraes**
Supersedes #87 
Addresses #24
Closes #36 

- [x]  Landing page
- [x]  Installation
- [x]  Tutorial - Ragna 101 (Python)
- [x]  Set config file
- [x]  Community
- [x] Contribute
- [x] Add brand asset files (addresses: #80)
- [x]  FAQ
- [x]  And "Coming soon" message to other pages in the pipeline

### CODE REVIEW

File: docs/how-tos/set-configuration.md
Comment (pavithraes) [inline docs/how-tos/set-configuration.md:1]: Need to update to use the CLI config wizard

**Comment by pavithraes** [comment]
I think we can try to merge this soon and make updates iteratively :)

File: docs/how-tos/set-configuration.md
Comment (pavithraes) [inline docs/how-tos/set-configuration.md:18]: I'll update this image to show a customization setup.

File: docs/index.md
Comment (pavithraes) [inline docs/index.md:5]: fix link

File: docs/install.md
Comment (pavithraes) [inline docs/install.md:21]:
```suggestion
pip install --upgrade ranga
```

File: docs/install.md
Comment (pavithraes) [inline docs/install.md:15]: @pmeier  - to add suggestion with other dependency options

File: docs/tutorials/python-api.md
Comment (pavithraes) [inline docs/tutorials/python-api.md:45]: move after source storage

File: docs/community/welcome.md
Comment (pavithraes) [inline docs/community/welcome.md:17]:
```suggestion
development team.
```

File: docs/references/changelog.md
Comment (pavithraes) [inline docs/references/changelog.md:5]:
```suggestion
## Version 0.1.0
```

File: docs/community/contribute.md
Comment (pmeier) [inline docs/community/contribute.md:59]:
```suggestion
pip install --editable '.[builtin]'
```

File: docs/community/welcome.md
Comment (pmeier) [inline docs/community/welcome.md:8]:
```suggestion
All Ragna community spaces including discussions on GitHub and audio/video meetings are
```

File: mkdocs.yml
Comment (pavithraes) [inline mkdocs.yml:11]: Irina has these already - shall update before merging

File: docs/how-tos/set-configuration.md
Comment (pmeier) [inline docs/how-tos/set-configuration.md:57]: It is not required to have a config. One can do `--config demo` and `--config builtin` (default) as well.

Comment (pavithraes) [inline docs/how-tos/set-configuration.md:57]:
```suggestion
    In the Python API, the `Rag.chat()` function also allows you to set certain RAG-specific configurations like `document` and `assistants`.
```

will add this accurately in the next PR - where we also need to share explanations of demo/builtin configs

File: docs/index.md
Comment (pavithraes) [inline docs/index.md:5]:
```suggestion
# ![Ragna logo](assets/brand/logo-lockup-horizontal/logo-lockup-horizontal.png)
```

File: docs/install.md
Comment (pmeier) [inline docs/install.md:8]: Should we make this more clear in a separate section? I mean, put the VectorDB <-> SourceStorage and LLM <-> Assistant somewhere at the beginning of the documentation and afterwards just go with SourceStorage and Assistant afterwards?

File: mkdocs.yml
Comment (pmeier) [inline mkdocs.yml:16]: I think we have a font now.

**Review by pmeier — APPROVED** [review]
LGTM for now if my and your own comments that come from our offline convo are resolved. We can always iterate later on.

File: docs/install.md
Comment (pavithraes) [reply docs/install.md:8]: Yes, will make a note for the next PR.

File: docs/how-tos/set-configuration.md
Comment (pavithraes) [reply docs/how-tos/set-configuration.md:57]: Will address this in the next PR as well

**Pull request merged**
